library/src/main/java/me/grantland/widget/AutofitHelper.java

```diff
@@ -433,12 +432,31 @@ public AutofitHelper setEnabled(boolean enabled) {
 
             if (enabled) {
                 mTextView.addTextChangedListener(mTextWatcher);
-                mTextView.addOnLayoutChangeListener(mOnLayoutChangeListener);
+
+                if ( android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.HONEYCOMB) {
+                    if (mOnLayoutChangeListener == null){
+                        mOnLayoutChangeListener = new View.OnLayoutChangeListener() {
```

Comment:
This is just a lazy initialization, there is no need for this instance in older versions...